apps/examples/webserver/webserver_main.c

```diff
@@ -111,25 +112,27 @@ static int http_send_chunk(struct http_client_t *client)
 	memset(msg_body, 'a', MAX_DATA_SIZE);
 	msg_body[MAX_DATA_SIZE] = 0;
 
-	ret = http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+	ret = http_send_response_chunk(client, HTTP_200_RESPONSE, "OK", msg_body, MAX_DATA_SIZE, NULL, FIRST_DATA);
```

Comment:
to demonstrate send data in binary format and including 0x00 char, 
will it not be better to send body len as MAX_DATA_SIZE + 1

Reply:
This code is not only for binary data but also for normal chunk response.